From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Daniel Muey <dan@cpanel.net>
Date: Tue, 22 Sep 2020 12:27:52 -0500
Subject: [PATCH 01/13] EA4 OBS ready

---
 build/php.m4       | 2 --
 sapi/fpm/config.m4 | 2 --
 2 files changed, 4 deletions(-)

diff --git a/build/php.m4 b/build/php.m4
index 9746ba2..75aea2c 100644
--- a/build/php.m4
+++ b/build/php.m4
@@ -1996,8 +1996,6 @@ dnl
 dnl Common setup macro for libxml.
 dnl
 AC_DEFUN([PHP_SETUP_LIBXML], [
-  PKG_CHECK_MODULES([LIBXML], [libxml-2.0 >= 2.9.0])
-
   PHP_EVAL_INCLINE($LIBXML_CFLAGS)
   PHP_EVAL_LIBLINE($LIBXML_LIBS, $1)
 
diff --git a/sapi/fpm/config.m4 b/sapi/fpm/config.m4
index 7c2fdec..088e52d 100644
--- a/sapi/fpm/config.m4
+++ b/sapi/fpm/config.m4
@@ -538,8 +538,6 @@ if test "$PHP_FPM" != "no"; then
     [no])
 
   if test "$PHP_FPM_SYSTEMD" != "no" ; then
-    PKG_CHECK_MODULES([SYSTEMD], [libsystemd >= 209])
-
     AC_CHECK_HEADERS(systemd/sd-daemon.h, [HAVE_SD_DAEMON_H="yes"], [HAVE_SD_DAEMON_H="no"])
     if test $HAVE_SD_DAEMON_H = "no"; then
       AC_MSG_ERROR([Your system does not support systemd.])
